frontend/src/pages/account-settings/components/login/components/update-logins-form/update-logins-form.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button, Input, Modal } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppForm, useCallback } from '~/libs/hooks/hooks.js';
+import {
+  type UserAuthResponse,
+  type UserUpdateLoginRequestDto,
+  userUpdateLoginValidationSchema,
+} from '~/packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  user: UserAuthResponse;
+  onSubmitUpdateUserLogin: (payload: UserUpdateLoginRequestDto) => void;
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const UpdateLoginsForm: React.FC<Properties> = ({
```

Comment:
```suggestion
const UpdateLoginForm: React.FC<Properties> = ({
```
don't forget about file name